easybuild/easyconfigs/p/PEAR/PEAR-0.9.11-foss-2018a.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'PEAR'
+version = '0.9.11'
+
+homepage = 'http://sco.h-its.org/exelixis/web/software/pear/'
+description = """PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. 
+ It is fully parallelized and can run with as low as just a few kilobytes of memory."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+# Needs manual download via web-form,
+# https://www.h-its.org/en/research/sco/software/#NextGenerationSequencingSequenceAnalysis
+# source_urls = ['http://sco.h-its.org/exelixis/web/software/pear/files/']
+sources = ['%(namelower)s-src-%(version)s.tar.gz']
+checksums = ['94f4a1835cd75ec6fab83405c2545ddba6b6bb1644579222e9cc2ad57a59d654']
```

Comment:
Hmm, I'm getting a different checksum here @verdurin (`e7b33c6f5e0281bdbe1362da3ae79e75c4f872f73818c23992f3e5746e555a79`).

Are the tarballs fingerprinted maybe?

Reply:
@boegel Could be. I did find that copying the URL from my e-mail didn't work. I had to open it in a browser, then copy over.

Reply:
They may well be generating it afresh each time? Even though that's nonsense.

Reply:
Can you try re-downloading? Do you still get the original checksum (or mine)?

Reply:
@boegel I've seen the same checksum three times.

Reply:
And I keep getting `e7b33c6f5e0281bdbe1362da3ae79e75c4f872f73818c23992f3e5746e555a79` as a checksum, even after requesting a new download link...

Were you using the same download link 3 times @verdurin?

Reply:
@boegel which file are you downloading? 
I tried again and got 94f4a1835cd75ec6fab83405c2545ddba6b6bb1644579222e9cc2ad57a59d654 for pear-src-0.9.11.tar.gz

Reply:
I looked into this again: if I copy-paste the URL in Firefox and download the tarball that way, I get a different checksum. Downloading with e.g. Chrome works fine though. Madness.